protocol/x/clob/keeper/liquidations_test.go

```diff
@@ -3829,7 +3828,9 @@ func TestGetPerpetualPositionToLiquidate(t *testing.T) {
 			},
 
 			expectedClobPair: constants.ClobPair_Btc,
-			expectedQuantums: constants.PerpetualPosition_OneTenthBTCLong.GetBigQuantums(),
+			expectedQuantums: new(big.Int).Neg(
+				constants.PerpetualPosition_OneTenthBTCLong.GetBigQuantums(),
+			),
```

Comment:
`GetPerpetualPositionToLiquidate` used to return position size in base quantums (not delta). 

The new `GetLiquidatablePositionSizeDelta` function returns delta, so flipping all the signs here.